### ISSUE TITLE
Bug: player head moves to bottom right corner of screen for no reason

### ISSUE THREAD

**Issue by sepsol** [MrCrayfish/Controllable#34]
Every time I try Controllable this happens in Minecraft. 

- Minecraft 1.14.4
- Forge 28.1.70
- OptiFine HD U F4
- Controllable-1.14.4-0.6.1

**Comment by SepSol** [comment]
I have a Bluetooth Xbox One S (1708) controller.

**Comment by MrCrayfish** [comment]
I don't know what you're talking about. Have you got a video? Maybe you need to adjust the dead zone in the settings

**Comment by SepSol** [comment]
I tested it again and I now suspect that it might be from my own controller. I'll update you if I get my hands on another controller and test it again.

**Comment by UnknownDev37** [comment]
Kinda have the same thing, Player looks down slowly, in menus its fine
In Minecraft 1.12.2 with Optifine HD U F5 using a program called Rainway to play on my phone, the "controller is showned as 
"ShanWan ps3/PC wired Gamepad" and it doesnt have any problems with other games.

**Comment by AmmoniumDichromate** [comment]
I believe I'm having a similar issue with my older xbox 360 wireless pad. I have to set a deadzone of at least 0.2 to compensate for the joystick drift (probably could be lower, but the menu cursor has some weird minimum speed that's quite high so a mild drift is amplified tenfold when navigating menus), when I lightly press my joystick in one of the four cardinal directions, the view will move diagonally rather than in that in the direction I'm pressing, however it'll straighten out when I press the joystick further.
The issue is amplified the higher I set the deadzone, if set to 0.9 than the view will go as far as moving left and right when pressing up and down on the joystick, If I set the deadzone to the minimum value than the issue goes away but of course the game becomes basically unplayable with the used controller.
